### PR TITLE
implement messages/ GET and POST

### PR DESCRIPTION
**Pull request by exterkamp**

### CODE REVIEW

File: packages/api/src/storage.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import {createLogger} from '../../shared/src/utils/logging'
+
+export class LocalApiStore {
+  _localFsPath: string
+  _messagesPath: string
+  constructor(localPath: string, folder?: string) {
+    folder = folder || 'App_File_Storage'
+    this._localFsPath = path.join(localPath, folder)
+    // Init the directories
+    fs.mkdirSync(path.join(this._localFsPath, 'messages'), {recursive: true})
```
Comment (exterkamp) [inline packages/api/src/storage.ts:14]: @patrickhulce does using `fs` inside of electron make sense, or is there some `electron-fs` package I should use.

Comment (patrickhulce) [reply packages/api/src/storage.ts:14]: oh no worries `fs` is totally fair game 👍 inside of our workers and main process you can think of it exactly like node

File: packages/api/src/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+## API Happy Hacking Hints
```
Comment (patrickhulce) [inline packages/api/src/README.md:1]: move this up one level so it's not in src?

File: packages/api/src/api.ts [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import express from 'express'
+import {v4 as uuidv4} from 'uuid'
 
 import type * as Api from '../../shared/src/utils/api'
+import {LocalApiStore} from '../src/storage'
```
Comment (patrickhulce) [inline packages/api/src/api.ts:5]: could just do `./storage` ;)

File: packages/api/src/storage.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    if (!fs.existsSync(this._messagesPath)) {
+      throw new Error('Not Found')
+    }
+
+    const file = JSON.parse(fs.readFileSync(this._messagesPath, 'utf8'))
+
+    return file.messages
+  }
+
+  postMessage(message: Api.Message) {
```
Comment (patrickhulce) [inline packages/api/src/storage.ts:42]: heh, `postMessage` :)

**Comment by patrickhulce** [comment]
thanks @exterkamp! 🎉

File: packages/api/src/api.ts (hunk 3 above)
Comment (exterkamp) [reply packages/api/src/api.ts]: whooops

**Comment by exterkamp** [comment]
Dunno how i made a `package-lock`, I think I ran an `npm` command on accident then remembered this used yarn...

**Pull request merged**